plugins/catalog/src/plugin.ts

```diff
@@ -177,6 +177,10 @@ export const EntityDependsOnResourcesCard = catalogPlugin.provide(
   }),
 );
 
+/**
+ * @deprecated This component is replaced by EntityCatalogGraphCard which is imported from backstage/catalog-graph-plugin. This component will be removed in an
```

Comment:
Package name should be `@backstage/plugin-catalog-graph` here as well

Reply:
Done👍